apollo-remote-state/client/src/operations/mutations/deleteTodo.tsx

```diff
@@ -0,0 +1,52 @@
+
+import { gql, useMutation } from "@apollo/client";
+import * as DeleteTodoTypes from './__generated__/DeleteTodo'
+
+export const DELETE_TODO = gql`
+  mutation DeleteTodo ($id: Int!) {
+    deleteTodo (id: $id) {
+      success
+      todo {
+        id
+        text 
+        completed
+      }
+      error {
+        ... on TodoNotFoundError {
+          message
+        }
+      }
+    }
+  }
+`
+
+export function useDeleteTodo () {
+  const [mutate, { data, error }] = useMutation<
+    DeleteTodoTypes.DeleteTodo, 
+    DeleteTodoTypes.DeleteTodoVariables
+  >(
+    DELETE_TODO,
+    {
+      update (cache, el) {
+        const deletedId = el.data?.deleteTodo.todo?.id
+        
+        cache.modify('ROOT_QUERY', {
```

Comment:
I know you and Ben talked about this internally, but I'll re-iterate the decision here. We're going to move away from demonstrating `cache.modify` wherever possible, as the learning Apollo journey is much easier to teach if we keep things as simple as possible, which means using `read/writeQuery` here. We'd like to keep `cache.modify` as a pro tool meant to be used by people who understand the cache internals.